projects/storyteller/webapp/proto/data.proto

```diff
@@ -56,18 +54,27 @@ message FileData {
   }
 }
 
+message Screenshot {
+  string filename = 1;
+  bytes screenshot = 2;
+}
+
 message StoryItem {
+  reserved 2;
   int64 timeMs = 1;
-  bytes screenshot = 2;
   string oneliner = 3;
   string note = 4;
+  string screenshot_filename = 5;
+  string id = 6;
 }
 
 message Story {
   int64 startTimeMs = 1;
```

Comment:
proto can be written in kebab_case only.
```
s/startTimeMs/start_time_ms
```
It just won't work with other cases.

Reply:
done